src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py

```diff
@@ -467,3 +468,448 @@ def _test_flexible_server_replica_mgmt(self, database_engine, resource_group):
+    @live_only()
+    def test_mysql_flexible_server_vnet_mgmt_supplied_vnet(self, resource_group):
+        self._test_flexible_server_vnet_mgmt_supplied_vnet('mysql', resource_group)
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(location=postgres_location)
+    @live_only()
+    def test_postgres_flexible_server_vnet_mgmt_supplied_vname_and_subnetname(self, resource_group):
+        self._test_flexible_server_vnet_mgmt_supplied_vname_and_subnetname('postgres', resource_group)
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(location=mysql_location)
+    @live_only()
+    def test_mysql_flexible_server_vnet_mgmt_supplied_vname_and_subnetname(self, resource_group):
+        self._test_flexible_server_vnet_mgmt_supplied_vname_and_subnetname('mysql', resource_group)
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(location=postgres_location, parameter_name='resource_group_1')
+    @ResourceGroupPreparer(location=postgres_location, parameter_name='resource_group_2')
+    @live_only()
+    def test_postgres_flexible_server_vnet_mgmt_supplied_subnet_id_in_different_rg(self, resource_group_1, resource_group_2):
+        self._test_flexible_server_vnet_mgmt_supplied_subnet_id_in_different_rg('postgres', resource_group_1, resource_group_2)
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(location=mysql_location, parameter_name='resource_group_1')
+    @ResourceGroupPreparer(location=mysql_location, parameter_name='resource_group_2')
+    @live_only()
+    def test_mysql_flexible_server_vnet_mgmt_supplied_subnet_id_in_different_rg(self, resource_group_1, resource_group_2):
+        self._test_flexible_server_vnet_mgmt_supplied_subnet_id_in_different_rg('mysql', resource_group_1, resource_group_2)
+
+    def _test_flexible_server_vnet_mgmt_supplied_subnetid(self, database_engine, resource_group):
+
+        # flexible-server create
+        if self.cli_ctx.local_context.is_on:
+            self.cmd('local-context off')
+
+        if database_engine == 'postgres':
+            sku_name = 'Standard_D2s_v3'
+            location = self.postgres_location
+        elif database_engine == 'mysql':
+            sku_name = 'Standard_B1ms'
+            location = self.mysql_location
+
+        vnet_name = 'clitestvnet'
+        address_prefix = '10.0.0.0/16'
+        subnet_name_1 = 'clitestsubnet'
+        subnet_prefix_1 = '10.0.0.0/24'
+        vnet_name_2 = 'clitestvnet1'
+        subnet_name_2 = 'clitestsubnet1'
+        # flexible-servers
+        servers = [self.create_random_name(SERVER_NAME_PREFIX, SERVER_NAME_MAX_LENGTH),
+                   self.create_random_name('azuredbcli', SERVER_NAME_MAX_LENGTH)]
+
+        # Case 1 : Provision a server with supplied Subnet ID that exists, where the subnet is not delegated
+
+        # create vnet and subnet
+        vnet_result = self.cmd('network vnet create -n {} -g {} -l {} --address-prefix {} --subnet-name {} --subnet-prefix {}'
```

Comment:
Could you use VirtualNetworkPreparer() as shown here https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/dls/tests/latest/test_dls_commands.py#L308?

Reply:
Ended up removing all the dependencies for live-only .